bibdk.make

```diff
@@ -108,11 +108,6 @@ projects[microcurl][download][type] = git
 projects[microcurl][download][url] = git@github.com:DBCDK/microcurl.git
 projects[microcurl][download][tag] = 7.x-1.1
 
-projects[open_adhl][type] = module
```

Comment:
Du kan ikke fjerne et modul som skal deaktiveres / afinstalleres. Drupal skal bruge modulet for at gøre det